src/BaGetter.Web/Pages/Package.cshtml.cs

```diff
@@ -208,6 +212,17 @@ private static List<VersionModel> ToVersions(IReadOnlyList<Package> packages, Nu
         return new HtmlString(readmeHtml);
     }
 
+    private HtmlString ParseReleaseNotes()
+    {
+        if (string.IsNullOrWhiteSpace(Package.ReleaseNotes))
+        {
+            return null;
+        }
```

Comment:
I don't like this very much. Saying a `HtmlString` (object) will be returned and then a `null` gets returned instead of a `string.Empty`.